.github/workflows/erlang.yml

```diff
@@ -0,0 +1,30 @@
+name: Erlang CI
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        otp: [21.3, 22.3, 23.3]
+    container:
+      image: erlang:${{matrix.otp}}-alpine
```

Comment:
```suggestion
```

Reply:
You need to remove this otherwise you're trying to install Erlang inside and Erlang image and not directly on top of Ubuntu.